docs/faq.rst

```diff
@@ -44,7 +44,7 @@ translations (you can do it in :guilabel:`Repository maintenance` in the
 on).
 
 If you've already ran into a merge conflict, the easiest way is to solve all
-conflicts locally at your workstation - is to simply add Weblate as a remote
+conflicts locally at your workstation : simply add Weblate as a remote
```

Comment:
This reverts text before 9531c2a4920c7293bbf44acb0389f86f80d8ac0e (see https://github.com/WeblateOrg/weblate/pull/3178) by @comradekingu

Reply:
That should have been an em dash. The colon is an error to.
Original sentence with a comma works.

Reply:
So : "the easiest way **is to** solve all conflicts locally on your machine, **is to** add Weblate as a remote"?
That sentence seems incorrect to me, but I'm not a native speaker